pkgs/applications/audio/real_time_config_quick_scan/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, fetchFromGitHub, perlPackages, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  pname = "realTimeConfigQuickScan";
+  version = "unstable-2020-08-03";
+
+  src = fetchFromGitHub {
+    owner  = "raboof";
+    repo   = pname;
+    rev    = "4b482db17f8d8567ba0abf33459ceb5f756f088c";
+    sha256 = "00l69gzwla9gjv5kpklgxlwnl48wnh8h6w0k8i69qr2cxigg4rhj";
+  };
+
+  buildInputs = [ perlPackages.perl makeWrapper ];
+
+  dontBuild = true;
+
+  installPhase = ''
+    mkdir -p $out/bin
+    mkdir -p $out/share/doc
+    # Install Script Files:
+    # *.pm files
+    for i in *.pm; do
```

Comment:
It would be also nice if you asked upstream to add an installation script so that distros do not need to write code like this:

https://src.fedoraproject.org/rpms/realTimeConfigQuickScan/blob/30c2ded31bfa26cd861d67fc32379484c1e1d58a/f/realTimeConfigQuickScan.spec#_43

https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=realtimeconfigquickscan-git&id=e9339a57f0e79ebb3526a5aae73040a11acea9fa#n25

It could be a simple makefile or maybe one of the things suggested in https://perlmaven.com/makefile-pl-of-module-install

Reply:
Thanks for providing most the text for [the issue](https://github.com/raboof/realtimeconfigquickscan/issues/37)!